sequence_tagging_for_ner/ner_final.py

```diff
@@ -0,0 +1,241 @@
+import math
+import gzip
+import paddle.v2 as paddle
+import paddle.v2.evaluator as evaluator
+import conll03
+import itertools
+
+word_dict, label_dict = conll03.get_dict()
+word_dict_len = len(word_dict)
+label_dict_len = len(label_dict)
+
+word_dim = 50
+caps_dim = 5
+context_length = 5
+hidden_dim = 300
+
+mix_hidden_lr = 1e-3
+default_std = 1 / math.sqrt(hidden_dim) / 3.0
+emb_para = paddle.attr.Param(
+    name='emb', initial_std=math.sqrt(1. / word_dim), is_static=True)
+std_0 = paddle.attr.Param(initial_std=0.)
+std_default = paddle.attr.Param(initial_std=default_std)
+
+
+def d_type(size):
+    return paddle.data_type.integer_value_sequence(size)
+
+
+def ner_net():
+    word = paddle.layer.data(name='word', type=d_type(word_dict_len))
+    #ws = paddle.layer.data(name='ws', type=d_type(num_ws))
+
+    word_embedding = paddle.layer.mixed(
+        name='word_embedding',
+        size=word_dim,
+        input=paddle.layer.table_projection(input=word, param_attr=emb_para))
+    #ws_embedding = paddle.layer.mixed(name='ws_embedding', size=caps_dim, 
+    #                    input=paddle.layer.table_projection(input=ws))
+    emb_layers = [word_embedding]  #[word_embedding, ws_embedding]
+
+    word_caps_vector = paddle.layer.concat(
+        name='word_caps_vector', input=emb_layers)
+    hidden_1 = paddle.layer.mixed(
+        name='hidden1',
+        size=hidden_dim,
+        bias_attr=std_default,
+        input=[
+            paddle.layer.full_matrix_projection(
+                input=word_caps_vector, param_attr=std_default)
+        ])
+
+    lstm_para_attr = paddle.attr.Param(initial_std=0.0, learning_rate=0.1)
+    hidden_para_attr = paddle.attr.Param(
+        initial_std=default_std, learning_rate=mix_hidden_lr)
+
+    lstm_1_1 = paddle.layer.lstmemory(
```

Comment:
The original configuration file uses simple RNN. By considering that we do not provide any examples of how to use the simple recurrent layer. I suggest not modify the original configuration file.